plugin/src/main/java/org/godotengine/openxr/vendors/GodotOpenXRHybridAppInternal.kt

```diff
@@ -30,69 +30,208 @@
 package org.godotengine.openxr.vendors
 
 import android.app.Activity
-import android.content.ComponentName
+import android.app.PendingIntent
 import android.content.Intent
 import android.util.Log
-import android.view.View;
+import android.view.View
 import org.godotengine.godot.Godot
+import org.godotengine.godot.GodotHost
+import org.godotengine.godot.GodotLib
 import org.godotengine.godot.plugin.GodotPlugin
 import org.godotengine.godot.plugin.UsedByGodot
+import org.godotengine.godot.utils.isHorizonOSDevice
 import org.godotengine.godot.utils.isNativeXRDevice
 
 /**
  * A plugin used internally for hybrid apps.
  */
 class GodotOpenXRHybridAppInternal(godot: Godot?) : GodotPlugin(godot) {
 	companion object {
-		protected val TAG = GodotOpenXRHybridAppInternal::class.java.simpleName
+		private val TAG = GodotOpenXRHybridAppInternal::class.java.simpleName
 
-		private const val IMMERSIVE_ACTIVITY = "com.godot.game.GodotApp"
-		private const val PANEL_ACTIVITY = "org.godotengine.openxr.vendors.GodotPanelApp"
+		private const val EXTRA_HYBRID_LAUNCH_DATA = "godot_openxr_vendors_hybrid_launch_data"
 
-		private const val IMMERSIVE_MODE = 0
-		private const val PANEL_MODE = 1
+		// TODO: Duplicate from the Godot Android library; should remove once the lib dependency
+		//  is updated
+		const val HYBRID_APP_PANEL_FEATURE = "godot_openxr_panel_app"
+		const val HYBRID_APP_PANEL_CATEGORY = "org.godotengine.xr.hybrid.PANEL"
+		const val HYBRID_APP_IMMERSIVE_CATEGORY = "org.godotengine.xr.hybrid.IMMERSIVE"
+		const val EXTRA_COMMAND_LINE_PARAMS = "command_line_params"
```

Comment:
This PR is still working with Godot 4.4, and these changes are coming (I think) in Godot 4.5. So, I think the comment here should say something about copying these in for compatibility with Godot 4.4, and that we can remove them when we update to depend on Godot 4.5+